### DISCUSSION TITLE
Welcome to awesome-ai-art-image-synthesis Discussions!

### DISCUSSION THREAD

**Discussion by altryne** [altryne/awesome-ai-art-image-synthesis#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jakowenko** [comment]
Tried to share an open-source project I created to generate AI art. Is that not allowed in "Show and tell"?
